live_pipeline/README.md

```diff
@@ -0,0 +1,9 @@
+# Live Pipeline
+
+This folder contains all code and resources required for the ETL of the live pipeline
+
+## Set-up and installation instructions
+
+1. Create venv `python3 -m venv venv`
+2. Activate venv `source .\venv\bin\activate`
+3. Install Requirements `pip install -r requirements.txt`
```

Comment:
What about the commands to run the program? Explicit is better than implicit for read me.